zhaquirks/xiaomi/aqara/light_acn.py

```diff
@@ -0,0 +1,87 @@
+from zigpy import types as t
+from zigpy.profiles import zha
+from zigpy.zcl.clusters.general import (
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.manufacturer_specific import ManufacturerSpecificCluster
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.xiaomi import BasicCluster, XiaomiCluster, XiaomiCustomDevice
+
+
+class OppleCluster(XiaomiCluster, ManufacturerSpecificCluster):
+    """Add Opple cluster for power outage memory attribute."""
+
+    cluster_id = 0xFCC0
```

Comment:
We should probably rename some of the clusters (and add `ep_attribute = opple_cluster` to the "E1 cluster"), but you should be able to use that "E1 cluster" and avoid specifying the `cluster_id` again:
```suggestion
class OppleCluster(XiaomiAqaraE1Cluster):
    """Add Opple cluster for power outage memory attribute."""

```
(+ add import)

Reply:
Makes sense. I changed the hierarchy so we have `XiaomiAqaraE1Cluster -> OppleCluster -> LightOppleCluster` now.

`OppleCluster` is here for the sake of semantics. Not all aqara devices implement Opple Cluster so it would be incorrect adding properties directly to `XiaomiAqaraE1Cluster`.

The name `XiaomiAqaraE1Cluster` is looks strange to me. What `E1` means? It is used in a number of devices so maybe it should be just `XiaomiAqaraCluster`?

Reply:
Opened https://github.com/zigpy/zha-device-handlers/pull/2667 for setting `ep_attribute` to `"opple_cluster"` in `XiaomiAqaraE1Cluster` (so all devices).
This is fine, as it's just the endpoint attribute used for accessing the cluster from ZHA or tests for example.
(`"aqara_cluster"` would be more correct, but quirks seem to have standardized upon using `opple_cluster`.)

> What E1 means?

It was initially the cluster used for newer Xiaomi E1 devices, but it's now also used by other devices. At the moment, we can't easily change the name, as we might break custom quirks. I'll keep this in mind, so we can change it in the future though.

Reply:
How about renaming `XiaomiAqaraE1Cluster` to `XiaomiAqaraCluster` and doing
`XiaomiAqaraE1Cluster = XiaomiAqaraCluster` to ensure backward compatibility with custom quirks?

Reply:
Keeping `XiaomiAqaraE1Cluster` should be fine for now, but I'll remember this if we do bigger changes in the future.